package.json

```diff
@@ -42,7 +42,7 @@
     "jest": "^26.0.1",
     "lint-staged": "^10.0.0",
     "mocha": "^8.0.1",
-    "nodemon": "^2.0.2",
+    "nodemon": "^1.8.1",
```

Comment:
Why?

Reply:
@ags1773 there is an issue with nodemon running in a loop